src/common/components/ui/OpenShiftVersionDropown.tsx

```diff
@@ -39,6 +43,7 @@ export const OpenShiftVersionDropdown = ({
   isDisabled,
   versions,
   getHelperText,
+  isOcmClient,
```

Comment:
This prop should be about "does the releases page need to be displayed", so it should be a boolean and the name should be something like `showReleasesPage`.

See the previous comment https://github.com/openshift-assisted/assisted-ui-lib/pull/1682/files/eece9aa0f1c0b4262e8220fef2014e68d12f40e7#r992316933

Reply:
Ok!